src/Dofs/MixedDofHandler.jl

```diff
@@ -153,6 +155,12 @@ function Base.push!(dh::MixedDofHandler, fh::FieldHandler)
     return dh
 end
 
+function _check_cellset_intersections(dh::MixedDofHandler, fh::FieldHandler)
+    for _fh in dh.fieldhandlers
+        isempty(intersect(_fh.cellset, fh.cellset)) || error("Each cell can only belong to a single FieldHandler.")
```

Comment:
```suggestion
        isdisjoint(_fh.cellset, fh.cellset) || error("Each cell can only belong to a single FieldHandler.")
```
Not important, but this is supported from Julia 1.5